src/CONST.ts

```diff
@@ -1201,6 +1201,12 @@ const CONST = {
         EXPENSIFY_EMAIL_DOMAIN: '@expensify.com',
     },
 
+    QBO_SELECTOR_OPTIONS: {
+        CROISSANT_CO_PAYROLL_ACCOUNT: 'CroissantCoPayrollAccount',
+        CROISSANT_CO_MONEY_IN_CLEARING: 'CroissantCoMoneyInClearing',
+        CROISSANT_CO_DEBTS_AND_LOANS: 'CroissantCoDebtsAndLoans',
```

Comment:
these are fake example values 😆 the real value comes from the `policy` object

Reply:
Yea, I just kept them as placeholders for now

Reply:
Not getting the values in policy yet

Reply:
@hayata-suenaga should i remove this now and do a blank implementation?

Reply:
yea, we need this PR to actually get the value from the backend, but we don't have to hold this PR for that 👍